src/templates/finops-hub/modules/dataExplorer.bicep

```diff
@@ -11,8 +11,14 @@ param location string = resourceGroup().location
 @description('Optional. Resource tags.')
 param tags object = {}
 
-@description('Required. The name of the Azure Key Vault instance.')
-param keyVaultName string
+// @description('Required. The name of the Azure Key Vault instance.')
+// param keyVaultName string
```

Comment:
Is there a reason you commented this out?

Reply:
Yea, causes validation errors as it isn't currently used. Not sure if we need it, so keeping for now.